src/bun.js/webcore/streams.zig

```diff
@@ -3184,8 +3172,7 @@ pub const FileSink = struct {
         return .{ .result = {} };
     }
 
-    pub fn flushFromJS(this: *FileSink, globalThis: *JSGlobalObject, wait: bool) JSC.Maybe(JSValue) {
-        _ = wait; // autofix
+    pub fn flushFromJS(this: *FileSink, globalThis: *JSGlobalObject, _: bool) JSC.Maybe(JSValue) {
```

Comment:
I think unused parameters should keep the variable name if comptime logic requires them. Makes it a little more clear what it would be used for. Otherwise the parameter should be completely removed from the function signature

Reply:
also interesting 
<img width="284" alt="Screenshot 2024-04-24 at 4 17 20 PM" src="https://github.com/oven-sh/bun/assets/35280289/f0d5c5fa-f60b-42b2-96e4-efbcc8263ff0">

Reply:
yeah